src/main/webapp/css/srp_style.css

```diff
@@ -11,6 +11,15 @@
   align-items: center;
 }
 
+.logo {
+  margin: 0 20px;
+  text-decoration: none;
+}
+
```

Comment:
So the searchbox here looks off center with respect to the login button. This confused me for a long time, but I figured it out - the &lt;form&gt; and &lt;table&gt; inside the searchbox have some bottom margin, and removing it causes it to center correctly.

You might also consider aligning the login button's height to the search button's height, as it currently looks visually off.

You might also consider putting some margin between the search button and the login button, as they're currently stuck together...

Reply:
I moved the button outside flexbox because I couldn't get the position to fix relative to the top right corner... Unsure if this is the best way though.

Reply:
Maybe not the way I'd do it, but this is fine.